Forms/index.html

```diff
@@ -9,82 +9,96 @@
-            <button type="submit" onclick="form.addEventListener()">Отправить</button><!--submit - все данные отпраятся, reset поля очищаются-->
-            <button type="button" onclick="clearAllFormInputs()">Очистить</button>
-        </div>
-        </form>
+                <p>Выберите ваш тип пива:
+                    <div>
+                        <input checked type="radio" name="beer" value="lager">Свелое
+                    </div>
+                    <div>
+                        <input  type="radio" name="beer" value="dark">Тёмное <!--checked - значение выборки по умолчанию-->
+                    </div>
+                    <div>
+                        <input type="radio" name="beer" value="bottling">Разливное
+                    </div>
+                    <div>
+                        <input type="radio" name="beer" value="nofilter">Не фильтрованное
+                    </div>
+                </p>
+            </div>
 
-        <div class="secound_collum">
-            <div>Текст </div>
-        </div>
+            <div class="form">
+                <p>Выберите любымый(ые) бренд(ы) пива</p>
+                <div>
+                    <input type="checkbox" name="box">Corona
+                </div>
+                <div>
+                    <input type="checkbox" name="box1">Alivaria
+                </div>
+                <div>
+                    <input type="checkbox" name="box2">Weihenstephaner
+                </div>
+                <div>
+                    <input type="checkbox" name="box3">Paulaner
+                </div>
+                <div>
+                    <input type="checkbox" name="box4">Жигулёвское
+                </div>
+            </div>
+            <div class="form">
+                Выберите фото вашего любимого пипа:<br>
+                <p>
+                    <input type="file" name="img"><!--Выбор файла-->
+                </p>
+            </div>
 
+            <div class="textarea">
+                <p>Опишите что-бы вы хотели добавить в разливное пиво</p>
+                <textarea name="text" cols="50" rows="10" placeholder="Впишите что нибудь )"></textarea>
+            </div>
+            <br>
+            <div class="select">
+                <p>Какого цвета бутылку вы предпочитаете</p>
+                <select name="color"> <!--выпадающий список, с помощью атрибута multiply можно сделать множественную выборку зажав ctcl -->
+                    <option value="red">Зеленая</option>
+                    <option selected  value="blue">Прозрачная</option><!--selected элемент показывается первым в списке-->
+                    <option value="green">Синяя</option>
+                    <option value="white">Коричневая</option>
+                </select>
+            </div>
+            <br>
+            <div class="select">
+                <button type="submit" onclick="form.addEventListener()">Отправить</button><!--submit - все данные отпраятся, reset поля очищаются-->
```

Comment:
What does "form.addEventListener()" do here?

Reply:
It's class in js I'm try to send info in other page